dirmon/dirmon/dirmon.stoq

```diff
@@ -18,7 +18,7 @@ Module = dirmon
 
 [Documentation]
 Author = Marcus LaFerrera
-Version = 3.0.0
+Version = 3.1.0
 Website = https://github.com/PUNCH-Cyber/stoq-plugins-public
 Description = Monitor a directory for newly created files for processing
 
```

Comment:
I think it would make sense to add a `min_stoq_version ` here, but can't until https://github.com/PUNCH-Cyber/stoq/issues/160 is tagged in a new release...  if the new release is 3.0.2, then I would add that here...